02_RegressionClassification/RegressionClassification.ipynb

```diff
@@ -0,0 +1,1734 @@
+{
```

Comment:
<p>Write out the form of the log model in words</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BNL-Fermi-Summer-School/tutorials_2024/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='75'/>